src/common/procedure/src/local.rs

```diff
@@ -405,7 +409,7 @@ impl LocalManager {
             exponential_builder: ExponentialBuilder::default()
                 .with_min_delay(self.retry_delay)
                 .with_max_times(self.max_retry_times),
-            store: ProcedureStore::new(self.state_store.clone()),
+            store: ProcedureStore::new(&self.parent_path, self.state_store.clone()),
```

Comment:
nitpick(non-blocking): maybe we can hold an `Arc<ProcedureStore>` and share it. Then `ProcedureStore` don't need to derive Clone.